### PR TITLE
Redirect legacy pulfa URIs to new pulfalight path

### PR DESCRIPTION
**Pull request by bess**
Connected to #639

### CODE REVIEW

**Review by tpendragon — CHANGES_REQUESTED** [review]
This is great! Can you add a test for collection IDs with periods in them?

See for example https://findingaids.princeton.edu/collections/MC001.02.01/c00003

Edit: Also, does this do a 301? Maybe a test for the code would be good too?

**Comment by bess** [comment]
@tpendragon Take another look when you get a chance?

File: spec/requests/catalog_request_spec.rb [hunk 1]
```diff
@@ -44,6 +44,19 @@
...
+  context "when requesting a URL from the legacy system" do
+    it "redirects requests for '/collections/eadid/componentid' to '/catalog/eadid_componentid'" do
+      get "/collections/C0140/c03411"
+      expect(response.code).to eq "301"
+      expect(response).to redirect_to("http://www.example.com/catalog/C0140_c03411")
+    end
+    it "redirects requests correctly when there are dots in the eadid" do
+      get "/collections/MC001.02.01/c00003"
+      expect(response.code).to eq "301"
+      expect(response).to redirect_to("http://www.example.com/catalog/MC001.02.01_c00003")
```
Comment (tpendragon) [inline spec/requests/catalog_request_spec.rb:56]: I'm real sorry to do this to you, and it's probably going to complicate things, but the actual URL is https://findingaids-beta.princeton.edu/catalog/MC001-02-01_c00003. I should have remembered, my bad.

**Review by tpendragon — CHANGES_REQUESTED** [review]

**Comment by bess** [comment]
@tpendragon Take another look?

**Review by tpendragon — APPROVED** [review]
This is great! Good work!

**Pull request merged**